content/pages/about/xsf/members.md

```diff
@@ -32,6 +32,7 @@ This page shows the elected members of the XMPP Standards Foundation, along with
 - David Banes
 - Mario Behling
 - Ludovic Bocquet
+- Jac-Carel Brand
```

Comment:
There seems to be a typo in the name. https://wiki.xmpp.org/web/JC_Brand_Application_2017